samplelistener/listener/listener.bal

```diff
@@ -1,24 +1,30 @@
 import ballerina/http;
 
-class SlackListener {
+class Listener {
    private http:Listener httpListener;
     private DispatcherService dispatcherService;
-   //private string verificationToken;
 
-   public isolated function init() returns error? { //receive verification token with listener config
-        self.httpListener = check new (8090); // param port
-        //self.verificationToken = userConfig.verificationToken
+   public isolated function init(int port = 8090) returns error? { 
```

Comment:
Didn't we discuss accepting an HTTP listener also?
 
```suggestion
   public isolated function init(int|http:Listener listenOn = 8090) returns error? { 
```

Reply:
Added, Thanks